Proposed AppBundle tip in upgrading_to_v3.md (#1131)
ezsystems/developer-documentation: docs/updating/upgrading_to_v3.md

#### PHP code and bundle organization

Since Symfony 4 `src/` code is no longer organized in bundles, `AppBundle` has been removed from the default eZ Platform install.
In order to adapt, you'll need to move all your PHP code, such as controllers or event listeners, to the `src` folder and use the `App` namespace for your custom code instead.

!!! tip "How to make AppBundle continue to work, for now"

>     Refactoring bundles for `src/` folder can involve quite some changes, if you'd like to make your `src/AppBundle` continue to work you can follow a guide on [Symfony Casts](https://symfonycasts.com/screencast/symfony4-upgrade/flex-composer.json#autoloading-src-amp-src-appbundle) on how to do that.

Reviewer: ```suggestion
    Refactoring bundles for `src/` folder can involve extensive changes, if you want to make your `src/AppBundle` continue to work, follow [an Autoloading src/ & src/AppBundle guide on Symfony Casts](https://symfonycasts.com/screencast/symfony4-upgrade/flex-composer.json#autoloading-src-amp-src-appbundle).
```

>     Or following [this guide on Symfony Casts](https://symfonycasts.com/screencast/symfony-bundle/extracting-bundle#using-a-path-repository), to create a [composer path repository](https://getcomposer.org/doc/05-repositories.md#path). If you have several bundles you can even move them into a `packages/` directory and load them all using:

Reviewer: ```suggestion
    You can also follow [Using a "path" Repository guide,](https://symfonycasts.com/screencast/symfony-bundle/extracting-bundle#using-a-path-repository) to create a [composer path repository.](https://getcomposer.org/doc/05-repositories.md#path)
    If you have several bundles you can move them into a `packages/` directory and load them all with:
    
```

    ```
    "repositories": [
        { "type": "path", "url": "packages/*" },
    ],
    ```
    

>     Once you are ready to refactor the code to `App` namespace, there is also [a chapter](https://symfonycasts.com/screencast/symfony4-upgrade/bye-appbundle) for that.

Reviewer: ```suggestion
    Once you are ready to refactor the code to `App` namespace, follow [Bye Bye AppBundle chapter.](https://symfonycasts.com/screencast/symfony4-upgrade/bye-appbundle)
```

#### PHP code and bundle organization

Since Symfony 4 `src/` code is no longer organized in bundles, `AppBundle` has been removed from the default eZ Platform install.
In order to adapt, you'll need to move all your PHP code, such as controllers or event listeners, to the `src` folder and use the `App` namespace for your custom code instead.

!!! tip "How to make AppBundle continue to work, for now"

>     Refactoring bundles for `src/` folder can involve extensive changes, if you want to make your `src/AppBundle` continue to work, follow [an Autoloading src/ & src/AppBundle guide on Symfony Casts](https://symfonycasts.com/screencast/symfony4-upgrade/flex-composer.json#autoloading-src-amp-src-appbundle).

Author: ```suggestion
    Refactoring bundles for `src/` folder can involve extensive changes, if you want to make your `src/AppBundle` continue to work, follow [an Autoloading src/AppBundle guide on Symfony Casts](https://symfonycasts.com/screencast/symfony4-upgrade/flex-composer.json#autoloading-src-amp-src-appbundle).
```

@MagdalenaZuba Afaik `src/` is already autoloaded by default in eZ Platform, so can simplify text here.
See: https://github.com/ezsystems/ezplatform/blob/master/composer.json#L104-L113

Reviewer: I just copied the title from the link, so the user will know from the start what to expect.

Reviewer: but I like the simplified version better, so yes, why not :)
